src/node/desktop/src/main/type-event-emitter.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * typed-event-emitter.ts
```

Comment:
minor detail that this file is named `type-event-emitter.ts` but it seems the intention was to name it `type`_d_`-event-emitter.ts`

Reply:
Oops, good eye!